weblogic-azure-vm/arm-oraclelinux-wls-admin/src/main/arm/nestedtemplates/adminTemplate.json

```diff
@@ -133,6 +133,54 @@
             "description": "Select appropriate VM Size as per requirement"
          }
       },
+      "virtualNetworkNewOrExisting": {
+         "type": "string",
+         "defaultValue": "new",
+         "allowedValues": [
+            "new",
+            "existing"
+         ],
+         "metadata": {
+            "description": "Specify whether to create a new or existing virtual network for the VM."
+         }
+      },
+      "virtualNetworkName": {
+         "type": "string",
+         "defaultValue": "jbosseap-vnet",
+         "metadata": {
+            "description": "Name of the existing or new VNET"
+         }
+      },
+      "virtualNetworkResourceGroupName": {
+         "type": "string",
+         "defaultValue": "[resourceGroup().name]",
+         "metadata": {
+            "description": "Resource group of Virtual network"
+         }
+      },
+      "addressPrefixes": {
+         "type": "array",
+         "defaultValue": [
+            "10.0.0.0/16"
+         ],
+         "metadata": {
+            "description": "Address prefix of the VNET."
+         }
+      },
+      "subnetName": {
+         "type": "string",
+         "defaultValue": "wls-subnet",
+         "metadata": {
+            "description": "Name of the existing or new Subnet"
+         }
+      },
+      "subnetPrefix": {
+         "type": "string",
+         "defaultValue": "10.0.0.0/24",
+         "metadata": {
```

Comment:
Are these CIDRs as big as they can be?

Reply:
Updated the default value to match the constrains in the UX